### ISSUE TITLE
playAnimation(int, int) yields wrong start and end progress

### ISSUE THREAD

**Issue by asuszek** [airbnb/lottie-android#491]
It looks like when LottieDrawable converts a frame number to progress, it divides the frame by the total number of frames:
 - https://github.com/airbnb/lottie-android/blob/master/lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java#L407
 - https://github.com/airbnb/lottie-android/blob/master/lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java#L455
 - https://github.com/airbnb/lottie-android/blob/master/lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java#L471

This is causing the progress to be smaller than expected. The frame should be divided by one less than the total frame count in order to have a progress in [0, 1] inclusive.

**Comment by gpeal** [comment]
@asuszek Why is that?
Let's say there are 2 frames.
If you say setMinFrame 0, 1, 2, you'd expect it to do 0/2, 1/2, and 2/2 right?

**Comment by asuszek** [comment]
I would expect if there are only 2 frames, your only options would be 0/1 and 1/1?

Admittedly, I haven't dug into the inner workings of how the animations are drawn, but my designer noted that everything in Lottie is vecotrized, including the motion, and the "frames" are just key frames to interpolate between. But if you're saying the second frame of a 2 frame animation should begin playing at 0.5, then I guess it's working as intended.

For some context, we found this bug because when trying to play just a static frame of a larger animation by doing playAnimation(187, 187), the frame it showed was always a bit before what we expected (and what the desktop program reported as frame 187). But then when using playAnimation(float, float) and calculating the progress the way I described in the first comment, it was dead on.
